### PR TITLE
View user subject

### PR DESCRIPTION
**Pull request by HuyenCherry**

### CODE REVIEW

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -12,3 +12,4 @@
 //= require summernote-init
 //= require activestorage
 //= require turbolinks
+//= require update_subject
```
Comment (danhmanh) [inline app/assets/javascripts/application.js]: new line

File: app/assets/javascripts/update_subject.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+$(document).on('turbolinks:load', function() {
+  $('.update-subject').on('click', function() {
+    console.log("aaa");
```
Comment (danhmanh) [inline app/assets/javascripts/update_subject.js]: xóa log đi chị

File: config/routes.rb [hunk 3]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  get 'user_subjects/new'
```
Comment (danhmanh) [inline config/routes.rb]: " "

File: app/controllers/subjects_controller.rb [hunk 4]
```diff
@@ -1,9 +1,15 @@
 class SubjectsController < ApplicationController
-  def show
+  def index
     @subject = Subject.all
```
Comment (danhmanh) [inline app/controllers/subjects_controller.rb]: `@subjects`

File: app/views/courses/index.html.erb [hunk 5]
```diff
@@ -1,86 +1,86 @@
...
 
-  <div class="title_right">
-  <div class="col-md-5 col-sm-5 col-xs-12 form-group pull-right top_search">
-    <div class="input-group">
-    <input type="text" class="form-control" placeholder="Search for...">
-    <span class="input-group-btn">
+    <div class="title_right">
+    <div class="col-md-5 col-sm-5 col-xs-12 form-group pull-right top_search">
+      <div class="input-group">
+      <input type="text" class="form-control" placeholder="Search for...">
```
Comment (danhmanh) [inline app/views/courses/index.html.erb]: i18n placeholder

Comment (itssomething) [reply app/views/courses/index.html.erb]: @danhmanh vl :)

**Pull request merged**